src/iter/mod.rs

```diff
@@ -2905,6 +2905,60 @@ pub trait ParallelIteratorExt: ParallelIterator {
     }
 }
 
+/// A thread pool backend that can process parallel iterators.
```

Comment:
s/process/execute/

Reply:
I'm interpreting this comment as "replace 'process' with 'execute'" but let me know if you meant something different